### PR TITLE
bedgraph: Add BED to bedGraph converter

### PR DESCRIPTION
**Pull request by JureZmrzlikar**

### CODE REVIEW

**Comment by tomazc** [comment]
@JureZmrzlikar: please, reformat the commit messages according to the [contributing guidelines](http://icount.readthedocs.io/en/latest/contributing.html#commit-and-pr-messages).

**Comment by JureZmrzlikar** [comment]
@tomazc , i fixed the commit messages. Currently, i added just `name` and `description` parameters into bedGraph header. Do you think it makes sense to also add any other ones? Which ones did you use in the "old" iCount?

File: iCount/files/bedgraph.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    header = [
+        'track',
+        'type=bedGraph',
+        'name="{}"'.format(name[:15]),
+        'description="{}"'.format(description[:60]),
+    ]
+
+    LOGGER.info('Converting %s to %s', bed, bedgraph)
+    with open(bedgraph, 'wt') as outfile:
+        print(' '.join(map(str, header)), file=outfile)
```
Comment (tomazc) [inline iCount/files/bedgraph.py]: @JureZmrzlikar: why use `print` instead of `.write` to write to files?

Comment (JureZmrzlikar) [reply iCount/files/bedgraph.py]: Why not? Isn't this is a matter of preference?
https://stackoverflow.com/questions/8598228/f-write-vs-print-f

Comment (tomazc) [reply iCount/files/bedgraph.py]: For consistency sake. Everywhere else we use `.write`. Can you please use it here as well.

Comment (JureZmrzlikar) [reply iCount/files/bedgraph.py]: Of course, no problem.

**Pull request merged**